### ISSUE TITLE
Install instructions: Spyder standalone installers

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-python#9264]
Spyder now offers standalone installers:
https://docs.spyder-ide.org/current/installation.html#standalone-installers-ref

I guess we should update our installation instructions and guide users to use the new installers?

cc @drammock

**Comment by drammock** [comment]
not until someone has tested them, and made sure it's possible to set the executable used in the built-in console to use the one in the mne/mnedev environment (or, if there is some new, simpler way to say "use this conda env")

**Comment by larsoner** [comment]
Closing in favor of #10148
